### DISCUSSION TITLE
[Bug]: makeInputSelect filter not working

### DISCUSSION THREAD

**Discussion by ken-ywk** [Power-Components/livewire-powergrid#762] [Show and tell]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### PowerGrid

3.5.12

### Laravel

9.40.1

### Livewire

2.10.7

### Alpine JS

3.10.2

### Theme

Tailwind 3.x

### Describe the bug.

I make a column with makeInputSelect, the filter is not working after select from the dropdown
```php   
         Column::make('CATEGORY 1', 'category_1_name', 'c1.name')
                ->sortable()
                ->searchable()
                ->makeInputSelect(
                    Category::where('level', 1)
                    ->where('status','active')
                    ->orderBy('name')
                    ->distinct()
                    ->select('name')
                    ->get(),
                    'name',
                    'c1.name'
                ),
```
the front end drop down do not have value

![image](https://user-images.githubusercontent.com/6763328/206830008-a8c9390a-e7b5-49ca-b5f0-2839dc78f492.png)

### To Reproduce...

First click on "FOO" then....

### Extra information

```markdown
<?php
 //...
```

**Comment by luanfreitasdev** [comment]
Hello!, I believe this happens because you limited the query. Maybe you should use` ->select('id', 'name') ` instead of -`>select('name')`?
